collect_app/src/main/java/org/odk/collect/android/utilities/ApplicationConstants.java

```diff
@@ -79,7 +79,6 @@ public abstract static class RequestCodes {
         public static final int CHANGE_SETTINGS = 27;
         public static final int MEDIA_FILE_PATH = 28;
 
-        public static final int FORMS_UPLOADED_NOTIFICATION = 97;
```

Comment:
If we were able to remove `FORMS_UPLOADED_NOTIFICATION` we probably could remove `FORMS_DOWNLOADED_NOTIFICATION` and `FORM_UPDATES_AVAILABLE_NOTIFICATION` too and use notificationIds instead?